modules/openy_features/openy_prgf/modules/openy_prgf_block_date/openy_prgf_block_date.info.yml

```diff
@@ -2,8 +2,7 @@ name: Open Y Paragraph for Date Block
 description: Provides a paragraph that renders Date block based on date and time (before, between, after).
 package: Open Y
 type: module
-core: 8.x
-version: 8.x-1.0
+core_version_requirement: ^8 || ^9
```

Comment:
looks like the version was removed accidentally

Reply:
I thought that too, but actually 
![image](https://user-images.githubusercontent.com/563412/92729407-f3333300-f37a-11ea-9e30-30ec9ee6ede0.png) @hamrant )))
This is *magic* of github review